src/format/svg/SVGData.hx

```diff
@@ -723,5 +737,28 @@ class SVGData extends Group {
     
 		return Std.parseInt ("0x" + hex);
 	}
-	
+
+	private static inline function parseRGBMatch(rgbMatch):Int
+	{
+			// CSS2 rgb color definition, matches 0-255 or 0-100%
+			// e.g. rgb(255,127,0) == rgb(100%,50%,0)
+
+			inline function range(val:Float):Int {
+				// constrain to Int 0-255
+        if (val<0) val = 0;

```

Comment:
Could you please use braces with `if` statements? It just makes the code slightly less error prone.